### PR TITLE
Update guides

### PR DESCRIPTION
**Pull request by wn96**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -18,6 +18,7 @@ endif::[]
 By: `Team F10-2`      Since: `Sep 2018`      Licence: `MIT`
 
 == Setting up
+This section describes the necessary dependencies required to install the application, together with the steps required to install.
```
Comment (FongYuan) [inline docs/DeveloperGuide.adoc]: We are not installing the application in the Dev Guide, rather we are setting up the tools necessary to work on the project's codebase.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -103,6 +104,7 @@ Having both Travis and AppVeyor ensures your App works on both Unix-based platfo
 When you are ready to start coding, you may move on to <<Design-Architecture>> to get some sense of the overall design.
 
 == Design
+This section depicts the implementation of the application, from UI to storage.
```
Comment (FongYuan) [inline docs/DeveloperGuide.adoc]: - 'Describes', not 'depicts'. 
- The meaning of "from UI to storage" is also unclear at this point in the dev guide. Can remove.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -647,7 +647,7 @@ From here, when the user decides to close the LoanBook, it will then loop throug
 
 == Documentation
 
-We use asciidoc for writing documentation.
+This section describe the way we document our project. We use asciidoc for writing documentation.
```
Comment (FongYuan) [inline docs/DeveloperGuide.adoc]: *describes

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -815,6 +817,8 @@ An alternative would be to use a regex to do string pattern matching on the adde
 
 == Dev Ops
 
+This sections provides a brief summery of the tools we use in the project, and how to integrate them to this project.
```
Comment (FongYuan) [inline docs/DeveloperGuide.adoc]: *summary

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -1250,16 +1246,26 @@ These instructions only provide a starting point for testers to work on; testers
...
 
 . Deleting a loan while all loans are listed
 
 .. Prerequisites: List all loans using the `list` command. Multiple loans in the list.
-.. Test case: `delete 1` +
+.. Test case: `delete i/1 x/a12345` +
    Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-.. Test case: `delete 0` +
+.. Test case: `delete i/1 x/a12345` +
+   Expected: No loan is deleted. Error details shown in the status message. Status bar remains the same.
```
Comment (FongYuan) [inline docs/DeveloperGuide.adoc]: Why isn't any loan deleted? Isn't the command input valid? What is the expected error message?

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -1250,16 +1246,26 @@ These instructions only provide a starting point for testers to work on; testers
...
+.. Test case: `delete i/1 x/wr0ngP4sS` +
```
Comment (FongYuan) [inline docs/DeveloperGuide.adoc]: Can add a note at the end "(wrong password)" to make it clear what is wrong. For all I know, the user really did set the password to `wr0ngP4sS`.

**Comment by wn96** [comment]
@FongYuan 

> The extra stuff on the right part of DeleteLoanSdForLogic.png also has incorrect formatting.

I'm not sure what is the issue. Do elaborate.

**Comment by FongYuan** [comment]
> @FongYuan
> 
> > The extra stuff on the right part of DeleteLoanSdForLogic.png also has incorrect formatting.
>
> I'm not sure what is the issue. Do elaborate.

deleteloansdforlogic

Refer to https://nus-cs2103-ay1819s1.github.io/cs2103-website/se-book-adapted/chapters/modeling.html#sequence-diagrams-basic for further help.

**Comment by prokarius** [comment]
@FongYuan: Perhaps we should allow the merger of this branch. We can make a separate PR for the diagrams. This will allow the other members to work on their enhancements to the guides as well.

**Comment by FongYuan** [comment]
For https://github.com/CS2103-AY1819S1-F10-2/main/issues/141

**Pull request merged**